openstack/baremetal/v1/nodes/requests.go

```diff
@@ -568,7 +568,7 @@ type LogicalDisk struct {
 	Controller string `json:"controller,omitempty"`
 
 	// A list of physical disks to use as read by the RAID interface.
-	PhysicalDisks []string `json:"physical_disks,omitempty"`
+	PhysicalDisks interface{} `json:"physical_disks,omitempty"`
```

Comment:
What do you think about `interface{}` and `[]interface{}`?
`interface {}` is easy to use, but `[] interface {}` can make the input must be a list.

Reply:
`[]interface{}` would be more strict and that's more appealing than a generic `interface{}`. Sure, let's go with that.